Symfony/CS/Tests/Fixer/PSR2/EofEndingFixerTest.php

```diff
@@ -52,17 +52,46 @@ public function provideCases()
 ',
             ),
             array(
-                "<?php\r\$a = 1;\n",
-                "<?php\r\$a = 1;",
+                "<?php\r\$a = 4;\n",
+                "<?php\r\$a = 4;",
             ),
             array(
-                "<?php\r\$a = 1;\n",
-                "<?php\r\$a = 1;\r",
+                "<?php\r\$a = 5;\n",
+                "<?php\r\$a = 5;\r",
```

Comment:
Ending in `\r\n` or `\n` should be allowed since we don't want to end up with a file with windows line endings then with a unix ending at the end...